README.md

```diff
@@ -319,6 +281,25 @@ This setting only applies if `verify` is set to `true`.
     verify-cert-email: john.smith@example.com
 ```
 
+### `verify-oidc-issuer`
```

Comment:
We only support getting the GitHub Actions OIDC token OR supplying our own token. Querying the OIDC issuer and getting a token in real time is NOT supported so let's make that clear in the documentation.